docs/guides/admin/docs/configuration/basic.md

```diff
@@ -3,106 +3,102 @@ Basic Configuration
 
 <!-- _Guide says nothing about how to start running OC, nor does it point to the "Next Step"
 as the Installation Docs do (i.e. Install doc refers to Config doc) -->
-This guide will help you to change the basic configuration settings which are required or at least strongly recommended
-for each Opencast installation. This is basically what you should do, right after installing Opencast on your machine.
+This guide will help you to change the basic configuration settings required or at least strongly recommended for each
+Opencast installation. It is basically what you should do, right after installing Opencast on your machine.  There are
+alternatives for some of these settings but this is the recommended setup.
 
-<!-- _"In most cases, ..." Unclear: the cases being the different Operating Systems? -->
-All settings changes are made to files residing in the Opencast configuration directory. In most cases, that should be either
-`/etc/opencast/` or `/opt/opencast/etc/`. Edit the files using the editor of your choice, e.g.:
-
-    vim /etc/opencast/custom.properties
+All settings changes are made to files residing in the Opencast configuration directory.
+The location of the configuration directory depends on how you installed Opencast.
+If you used the Linux packages, the location is `/etc/opencast`.
 
 
 Step 1: Setting the Server URL
 ------------------------------
 
-By default, only connections from the local machine are accepted by Opencast.  You want to change this if the system
-should be accessible within a network.
-
-First, find the property `org.opencastproject.server.url` in your `custom.properties` configuration file and set it to
-your own domain name:
+First, find the property `org.opencastproject.server.url` in `etc/custom.properties` and set your domain name.
```

Comment:
suggest taking out "First", or adding "Next/Then/After that" to later steps. otherwise this feels out of place.